src/App.js

```diff
@@ -1,28 +1,57 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
+import React, {Component} from 'react';
 import './App.css';
+import TodoInput from './components/TodoInput/TodoInput';
+import TodoList from "./components/TodoList"
+import TodoItemObj from "./classes/TodoItemObj";
 
 class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
-    );
-  }
+    constructor(props) {
+        super(props);
+        this.state = {
+            todoArray: [],
+        };
+
+        this.addTodo = this.addTodo.bind(this);
+        this.changeTodo = this.changeTodo.bind(this);
+    }
+
+    addTodo(todoTitle) {
+        todoTitle && this.setState({
+            todoArray: this.state.todoArray.concat(new TodoItemObj(todoTitle))
```

Comment:
այստեղ կարող էիր `push` օգտագործել, քանի որ միայն մի անդամ ես ավելացնում

Reply:
https://stackoverflow.com/questions/37435334/correct-way-to-push-into-state-array/43003547

NEVER mutate this.state directly, as calling setState() afterwards may replace the mutation you made. Treat this.state as if it were immutable.

Reply:
չէ ես դա նկատի չունեի 
էլի պետք է `setState` օգտագործել , միայն concat ի փոխարել օգտագործել push